Emby.AutoOrganize/PluginEntryPoint.cs

```diff
@@ -71,13 +78,13 @@ public Task RunAsync()
             Current = this;
             FileOrganizationService = new FileOrganizationService(_taskManager, _repository, _logger, _libraryMonitor, _libraryManager, _config, _fileSystem, _providerManager);
 
-            FileOrganizationService.ItemAdded += _organizationService_ItemAdded;
-            FileOrganizationService.ItemRemoved += _organizationService_ItemRemoved;
-            FileOrganizationService.ItemUpdated += _organizationService_ItemUpdated;
-            FileOrganizationService.LogReset += _organizationService_LogReset;
+            FileOrganizationService.ItemAdded += OrganizationService_ItemAdded;
+            FileOrganizationService.ItemRemoved += OrganizationService_ItemRemoved;
+            FileOrganizationService.ItemUpdated += OrganizationService_ItemUpdated;
+            FileOrganizationService.LogReset += OrganizationService_LogReset;
```

Comment:
The naming we use is: On[PropertyName][Action], so in this case: `OnOrganizationServiceLogReset`